### ISSUE TITLE
winPB: Make removal of speech mark in %PATH it's own task in Cygwin

### ISSUE THREAD

**Issue by Willsparker** [AdoptOpenJDK/openjdk-infrastructure#1401]
ping @lumpfish 

Currently, this task: https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/3143979688df2c97ea6b41347feaef8438809885/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml#L113

Will remove the stray speech mark in the Windows path before then adding cygwin's folder to the front of the path, however on some Windows systems this errors, with the following error message:

```
TASK [cygwin : Add c:\cygwin64\bin to front of %PATH%] *************************
fatal: [IP]: FAILED! => {"changed": true, "cmd": "setx /M PATH \"C:\\cygwin64\\bin;%PATH:\\\"=%\"", "delta": "0:00:00.218777", "end": "2020-06-17 01:27:23.008818", "msg": "non-zero return code", "rc": 1, "start": "2020-06-17 01:27:22.790040", "stderr": "ERROR: Invalid syntax. Default option is not allowed more than '2' time(s).\r\nType \"SETX /?\" for usage.\r\n", "stderr_lines": ["ERROR: Invalid syntax. Default option is not allowed more than '2' time(s).", "Type \"SETX /?\" for usage."], "stdout": "", "stdout_lines": []}
```
Running the command that is written in the task on the windows machine itself also didn't work, however running this manually did: `set NEWPATH=%PATH:"=% && setx /M PATH "C:\cygwin64\bin;%NEWPATH%"`  which can be 'ansibilised' as two separate tasks. This has the side benefit of being much clearer to read and understand what's going on.

**Comment by Willsparker** [comment]
I've found a solution that would fix this issue- as in split out the tasks, as this makes it more readable for people who aren't as familiar with `cmd`/ Windows functions. I confirmed this does remove the speech marks as intended.
```
- name: Remove speech mark from the Path
  win_shell: 'setx /M PATH "%PATH:\"=%"'
  args:
    executable: cmd
  tags:
    - cygwin
```
I'm happy to put it in, the problem is that the error message I posted above would suggest there was/is an issue with the `setx` syntax - possibly something with the speech marks in the command. It should be mentioned that the error occurred on a machine that has had the Cygwin task run on it before. I hypothesise the issue is because the syntax is correct when the speech mark is in the `%PATH%`, but then fails when it isn't - i.e. `%PATH%` has had the speech mark removed from the cygwin role running before. 

EGiven that the path won't need to be 're-updated' if the `cygwin` installation is skipped, I'm also going to add `when: (not cygwin_installed.stat.exists)` to the tasks that change the path. This should stop issues occurring if the playbook is ran on a machine the second time.
